catalyst-generator/src/main/resources/templates/usecase/iotibm/util/TokenReceiver.java

```diff
@@ -8,19 +8,20 @@
 
 import <%packageName%>.service.BlockchainConnector;
 import java.io.IOException;
+import javax.annotation.PostConstruct;
 import org.springframework.beans.factory.InitializingBean;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
 @Component
 public class TokenReceiver implements InitializingBean {
 
   private String token;
-  private BlockchainConnector blockchainConnector;
 
-  public TokenReceiver(BlockchainConnector blockchainConnector) {
-    this.blockchainConnector = blockchainConnector;
-  }
+  @Autowired
+  private BlockchainConnector blockchainConnector;
 
+  @PostConstruct
```

Comment:
@yurii-uhlanov use either InitializingBean or PostConstruct. Using both is not needed